telemetry/software/telemetry/app.py

```diff
@@ -0,0 +1,24 @@
+"""
+Entrypoint app file
+
+Objective: initialize and run telemetry software pipeline
+"""
+
+from library import (
+    ArduinoSerialIn,
+    RedisDataSender,
+)
+
+
+def run():
+    """
+    Run the data pipeline
+    """
+    stream = ArduinoSerialIn()
```

Comment:
can we do this stuff in a config file? https://docs.python.org/3/library/configparser.html